### PR TITLE
BMP-345 - style error message

### PR DESCRIPTION
**Pull request by Danielecina**
#### Linked Task
https://github.com/mia-platform/json-editor/pull/5

#### Checklist
- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [ ] tests are included
- [ ] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

#### Img
Schermata 2020-07-02 alle 15 40 45

### CODE REVIEW

File: packages/api-explorer/src/JsonForm/antd-theme-json-editor.js [hunk 1]
```diff
@@ -47,6 +47,17 @@ const antdTheme = (JSONEditor) => class extends JSONEditor.defaults.themes.boots
     el.style.display = 'inline-flex'
     return el
   }
+
+  addInputError (input, text) {
```
Comment (dibericky) [inline packages/api-explorer/src/JsonForm/antd-theme-json-editor.js:51]: This could be made inside json-editor with an option

Comment (dibericky) [reply packages/api-explorer/src/JsonForm/antd-theme-json-editor.js:51]: In this way you do not need to `return input` (that is made only in bootstrap4's method, but the interface should be equals in all themes)

File: packages/api-explorer/src/JsonForm/antd-theme-json-editor.js [hunk 2]
```diff
@@ -47,6 +47,17 @@ const antdTheme = (JSONEditor) => class extends JSONEditor.defaults.themes.boots
...
+    const el = super.addInputError(input, text)
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/JsonForm/antd-theme-json-editor.js:52]: This el is useless you should use the provided `input` as argument

File: packages/api-explorer/package.json [hunk 3]
```diff
@@ -6,7 +6,6 @@
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.1",
     "@fortawesome/fontawesome-free": "^5.13.0",
```
Comment (Danielecina) [inline packages/api-explorer/package.json:8]:
```suggestion
    "@fortawesome/fontawesome-free": "^5.13.0",
    "@mia-platform/json-editor": "^2.3.1",
```

File: packages/api-explorer/package-lock.json [hunk 4]
```diff
@@ -99,11 +99,6 @@
 				"unist-util-visit": "^1.3.0"
 			}
 		},
```
Comment (Danielecina) [inline packages/api-explorer/package-lock.json:101]:
```suggestion
		},
		"@mia-platform/json-editor": {	
			"version": "2.3.1",	
			"resolved": "https://registry.npmjs.org/@mia-platform/json-editor/-/json-editor-2.3.1.tgz",	
			"integrity": "sha512-fNi548xMCU9gRqqTA/qX5a1wREWtsMNbiHJ4lYJAMDKDKj8q49J1czLdQWi4oeNr353c4E22i0kxb5ZlLH3kbg=="	
		},
```

File: packages/api-explorer/src/JsonForm/index.jsx [hunk 5]
```diff
@@ -67,7 +62,8 @@ class JsonForm extends Component {
         theme: 'antdTheme',
         max_depth: 3,
         use_default_values: false,
-        remove_empty_properties: false
+        remove_empty_properties: false,
+        validation_error_placement: 'under_field'
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/JsonForm/index.jsx:66]: Wasn't it `below_field` ?

Comment (fredmaggiowski) [reply packages/api-explorer/src/JsonForm/index.jsx:66]:
```suggestion
        validation_error_placement: 'below_field'
```

Comment (Danielecina) [reply packages/api-explorer/src/JsonForm/index.jsx]: correct! done

**Pull request merged**